pypesto/petab/importer.py

```diff
@@ -320,16 +320,31 @@ def create_prediction(self,
         if objective is None:
             objective = self.create_objective()
 
+        # create a identifiers of preequilibration and simulation condition ids
+        # which can then be stored in the prediction result
+        edata_conditions = objective.amici_object_builder.petab_problem.\
+            get_simulation_conditions_from_measurement_df()
+        if 'preequilibrationConditionId' not in list(edata_conditions.columns):
+            preeq_dummy = [''] * edata_conditions.shape[0]
+            edata_conditions['preequilibrationConditionId'] = preeq_dummy
+        edata_conditions.drop_duplicates(inplace=True)
+        condition_ids = [
+            edata_conditions.loc[id, 'preequilibrationConditionId'] + '::' +
+            edata_conditions.loc[id, 'simulationConditionId']
```

Comment:
```suggestion
        if PREEQUILIBRATION_CONDITION_ID not in list(edata_conditions.columns):
            preeq_dummy = [''] * edata_conditions.shape[0]
            edata_conditions[PREEQUILIBRATION_CONDITION_ID] = preeq_dummy
        edata_conditions.drop_duplicates(inplace=True)
        condition_ids = [
            edata_conditions.loc[id, PREEQUILIBRATION_CONDITION_ID] + '::' +
            edata_conditions.loc[id, SIMULATION_CONDITION_ID]
```

with 
```python
from petab.C import PREEQUILIBRATION_CONDITION_ID, SIMULATION_CONDITION_ID
```

What is `'::'`? Should this be a constant?

Reply:
'::' is meant to be a separator, which can be used later to reconstruct the ids of the respective conditions. If one splits with respect to this, one gets more info about the simulations...

Reply:
Makes sense but then it should be a constant I guess? e.g. `CONDITION_DELIMITER = '::'`